submissions/Nemooochka/friends-app/js/filters.js

```diff
@@ -0,0 +1,111 @@
+import {Url} from "./url.js";
+
+export class Filters {
+    constructor(users, node) {
+        this.users = users;
+        this.node = node;
+        this.filterByGenderText = 'filterByGender';
+    }
+
+    getCardsChildNodesArr() {
+        let cardsChildNodes = this.node.childNodes;
+        return [...cardsChildNodes];
+    }
+
+    activateFilterByGenderIcon(filterValue) {
+        document.querySelector(`[filter-gender='${filterValue}']`).checked = true;
+    }
+
+    filterByName({target}) {
+        let inputNameValue = target.value.toLowerCase();
+
+        if (inputNameValue.length > 1) {
+            this.users.map((elem, index) => {
+                let userName = elem.name.first + ' ' + elem.name.last;
```

Comment:
use template strings instead of concatenation and add the lowercase method in this line as well